src/main/java/Ictium.java

```diff
@@ -1,10 +1,14 @@
 public class Ictium extends Core {
 
     @Override
-    public SplitResult split(int tmpInKelvin, int timeInSeconds) {
+    public SplitResult split(int tmpInKelvin, int timeInSeconds) throws MeltDownException {
         float steam = 40 * tmpInKelvin;
         float residualHeatInKelvin = timeInSeconds / 0.3f * 18;
 
+        if (tmpInKelvin > 150){
+            throw new MeltDownException("ICTIUM MELTDOWN, WE F*CKED UP AGAIN");

```

Comment:
Let tijdens het inleveren tijdens een toets of eindopdracht wel op je taalgebruik ⛔